ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -46,6 +47,18 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
     spec: { queries, thresholds, y_axis },
     contentDimensions,
   } = props;
+  const chartsTheme = useChartsTheme();
+
+  // TODO: consider refactoring how the layout/spacing/alignment are calculated
+  // the next time significant changes are made the the time series panel (e.g.
```

Comment:
nit:
```suggestion
  // the next time significant changes are made to the time series panel (e.g.
```

Reply:
hmm seems like it makes sense to move the legend to the TimeSeriesChart level then  🤔

Reply:
Yeah, I think there's a strong argument to move it down the next time we make any meaningful changes to the legend. I'll add a bit about that in this comment too.